libs/verifiable-credentials/src/main/java/uk/gov/di/ipv/core/library/verifiablecredential/domain/VerifiableCredentialStatus.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.di.ipv.core.library.verifiablecredential.domain;
+
+import lombok.Getter;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+@Getter
+public enum VerifiableCredentialStatus {
+    CREATED("created"),
+    PENDING("pending");
+    private static final Map<String, VerifiableCredentialStatus> STATUSES =
+            Arrays.stream(VerifiableCredentialStatus.values())
+                    .collect(
+                            (Collectors.toMap(
+                                    VerifiableCredentialStatus::getStatus, Function.identity())));
+    private final String status;
+
+    VerifiableCredentialStatus(String status) {
+        this.status = status;
```

Comment:
I'm guessing the plan is to update this to CREATED when the actual VC is received?

Reply:
There's no immediate plan to fixup the CRI response table when the actual VC is received. The CRI response item will hang around for its 14 day TTL and then gracefully disappear.